### PR TITLE
New post gui challenge split text

### LINKED ISSUE #5048: Thinking on ways to solve split text animations [4/14/2021]
* [Draft](https://docs.google.com/document/d/10CpBY0zzYK5IMtSe7VMLYIMr0xSb85kX_fEQX96ZdS8/edit#heading=h.yrry18w66spp)

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #5048

todo: 
- [ ] final reviews
- [x] add proper youtube link

### CODE REVIEW

**Comment by rachelandrew** [comment]
@argyleink let me know when this is ready for a final review

**Comment by argyleink** [comment]
It's ready!

File: src/site/content/en/blog/building-split-text-animations/index.md [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+tags:
+  - blog
+  - css
+  - dom
+  - javascript
+  - animation
+---
+
+In this post I want to share thinking on ways to solve split text animations and
+interactions for the web that is minimal, accessible and works across browsers.
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-split-text-animations/index.md:20]:
```suggestion
In this post I want to share thinking on ways to solve split text animations and
interactions for the web that are minimal, accessible, and work across browsers.
```

File: src/site/content/en/blog/building-split-text-animations/index.md [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+If you prefer video, here's a YouTube version of this post:
+
+{% YouTube '3hvN7bkjZBk' %}
+
+## Overview
+
+Split text animations can be amazing. We'll be barely scratching the surface of
+animation potential in this post, but it does provide a foundation to build
+upon. The goal is to animate progressively. The text should be readable by
+default, with the animation built on top of it. Split text motion effects can
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-split-text-animations/index.md:45]:
```suggestion
default, with the animation built on top. Split text motion effects can
```

File: src/site/content/en/blog/building-split-text-animations/index.md [hunk 3]
```diff
@@ -0,0 +1,562 @@
...
+transitions and animations, while the Javascript media query will be used to
+withhold the HTML manipulation. 
+
+{% Banner 'neutral' %} **Question:** What else should be used to withhold split
+text animations?{% endBanner %}
+
+
+#### Preparing the CSS conditional
+
+I used PostCSS to enable the syntax of Media Queries Level 5, where I can store
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-split-text-animations/index.md:104]:
```suggestion
I used PostCSS to enable the syntax of [Media Queries Level 5](https://drafts.csswg.org/mediaqueries-5/), where I can store
```

File: src/site/content/en/blog/building-split-text-animations/index.md [hunk 4]
```diff
@@ -0,0 +1,562 @@
...
+}
+```
+
+With the PostCSS custom property and a Javascript boolean, we're ready to
+conditionally upgrade the effect. That rolls us into the next section where I
+break down the Javascript for transforming strings into elements.
+
+## Splitting Text
+
+Text letters, words, lines, etc cannot be individually animated with CSS or JS.
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-split-text-animations/index.md:151]:
```suggestion
Text letters, words, lines, etc., cannot be individually animated with CSS or JS.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
just a couple of tiny fixes to do

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**